processing/src/main/java/org/apache/druid/query/timeseries/TimeseriesQuery.java

```diff
@@ -50,6 +50,12 @@
 {
   public static final String CTX_GRAND_TOTAL = "grandTotal";
   public static final String SKIP_EMPTY_BUCKETS = "skipEmptyBuckets";
+  // This context parameter is an undocumented parameter, used internally, to allow timeseries query with
```

Comment:
I think this could be clearer. How about:

> "timestampResultField" is an undocumented parameter used internally by the SQL layer. It is necessary because when the SQL layer generates a Timeseries query for a group-by-time-floor SQL query, it expects the result of the time-floor to have a specific name. That name is provided using this parameter.

Reply:
Sounds good. Done